hoomd/md/improper.py

```diff
@@ -110,3 +110,46 @@ def __init__(self):
                 chi0=hoomd.data.typeconverter.nonnegative_real,
                 len_keys=1))
         self._add_typeparam(params)
+
+class Periodic(Improper):
```

Comment:
Add this class name to the list in `module-md-improper.rst` so the documentation appears.